core/ecschema-metadata/src/test/SchemaKey.test.ts

```diff
@@ -137,4 +138,34 @@ describe("SchemaKey", () => {
     });
 
   });
+
+  describe("fromJson", () => {
+    let testKey2: SchemaKey;
+
+    it("should return a SchemaKey given a SchemaKeyProp using fromJson", async () => {
+      testKey2 = SchemaKey.fromJson({name: "testKey2", read: 1, write: 0, minor: 12});
+      expect(testKey2).to.not.eql(undefined);
+      expect(testKey2.name).to.eql("testKey2");
```

Comment:
Any reason not to test the other properties?

Reply:
No, not really. I just added more assertions that test the other properties.